public/js/views/management/pay-methods.jsx

```diff
@@ -35,6 +40,12 @@ export default class PayMethods extends Component {
   }
 
   render() {
+
+    var isDeleteDisabled = !this.props.payMethods ||
```

Comment:
Afaik this is ok because this is an unambiguosly incomplete statement.

E.g: http://jsbin.com/hiqizu/edit?js,console 

Interestingly babel does also flatten this out looking at the demo output https://babeljs.io/repl/#?experimental=true&evaluate=true&loose=false&spec=false&code=foo%20%3D%20false%20%7C%7C%0A%20%20%20%20%20%20true%3B%0A